bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVBridgeHandler.java

```diff
@@ -149,4 +156,12 @@ private void initiateConnexion() {
     public @Nullable PointType getLocation() {
         return locationProvider.getLocation();
     }
+
+    public TranslationProvider getI18nProvider() {
```

Comment:
Adding Getters for passing services to other services looks a little bit messy. Maybe we can get https://github.com/openhab/openhab-core/pull/1941 running to let the framework add additional service references to `ThingHandlerAction`s.

Reply:
This PR in core framework was apparently abandoned (opened almost one year ago with absolutely no feedback from core maintainers).

Reply:
@cweitkamp : I see 3 options:

1. someone reviews quickly the PR in core framework to allow 
2. we keep ThingHandlerService as it is now and the only solution I see is the one I implemented
3. we decide to not use ThingHandlerService in discovery services

WDYT ?

Reply:
> This PR in core framework was apparently abandoned (opened almost one year ago with absolutely no feedback from core maintainers).

Yes, I know. Please see my comment as reminder for myself to pick it up quickly. I see a lot of possibilities if it is working.

I will go for No. 1. We do not want to put a bad implementation into our code potentially picked by others afterwards. And discovery service is the biggest use case for it.

Reply:
@cweitkamp : is there some progress on your side? 
In case this PR is not merged quickly, I will have to undo the PR that introduced usage if @text for the discovery label.

Reply:
Unfortunately not. I tried to get https://github.com/openhab/openhab-core/pull/1941 running in my environment but it includes a breaking change which requires to fix nearly every add-on which implements a `ThingHandlerService`. Round about 200 classes to be changed.

Reply:
So are you ok to merge this PR or should I remove the internationalization of the discovery result ?